mon-pix/tests/acceptance/account-recovery/update-sco-record_test.js

```diff
@@ -0,0 +1,235 @@
+import { describe } from 'mocha';
+import { expect } from 'chai';
+import { setupApplicationTest } from 'ember-mocha';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import visit from '../../helpers/visit';
+import { currentURL, triggerEvent } from '@ember/test-helpers';
+import setupIntl from '../../helpers/setup-intl';
+import { clickByLabel } from '../../helpers/click-by-label';
+import { fillInByLabel } from '../../helpers/fill-in-by-label';
+import { contains } from '../../helpers/contains';
+import { Response } from 'ember-cli-mirage';
+
+describe('Acceptance | account-recovery | UpdateScoRecordRoute', function() {
+
+  setupApplicationTest();
+  setupMirage();
+  setupIntl();
+
+  context('when account recovery is disabled by default', function() {
+
+    it('should redirect to login page', async function() {
+      // given / when
+      await visit('/recuperer-mon-compte');
+
+      // then
+      expect(currentURL()).to.equal('/connexion');
+    });
+  });
+
+  context('when account recovery is enabled', () => {
+
+    it('should access account recovery page and show reset password form', async function() {
```

Comment:
**chipotage**
```suggestion
    it('should show reset password form', async function() {
```

Reply:
Je propose de le déplacer sous le contexte `and user clicks on email link`

Reply:
done